tools/data/hvu/generate_file_list.py

```diff
@@ -0,0 +1,151 @@
+import argparse
```

Comment:
Can this part be moved to (parse_file_list.py)[https://github.com/open-mmlab/mmaction2/blob/master/tools/data/parse_file_list.py] ?

Reply:
It would be hard to do so: 1. The format of the datalist of HVU dataset is specific and can only be handled with this piece of code. Besides, the code can not be used on any other dataset. 2. Since the directory `tools` is not under the mmaction2 repository, I can not reuse the code in `parse_file_list.py` (You see that I copied the function `parse_directory` into this file). Thus I think it would be better to keep this code in the `hvu` directory.

Reply:
All right. But can we use `from ..parse_file_list import parse_directory` to avoid rewriting the `parse_directory`

Reply:
When executing the import `from ..parse_file_list import parse_directory`, python will throw the ValueError: attempted relative import beyond top-level package .

Reply:
Maybe we can try `sys.path.append` ?

Reply:
Is there any walkaround to avoid adding 3rdparty to setup.cfg? If so, it will be OK. (I have tried isort:skip and it didn't work). Using sys.path.append is OK. Howevere, the problem is: if u try to `import  parse_file_list`, 3rdparty will be added.

Reply:
All right